src/helm/benchmark/scenarios/test_math_scenario.py

```diff
@@ -1,9 +1,14 @@
+import pytest
 from tempfile import TemporaryDirectory
 
 from helm.benchmark.scenarios.math_scenario import MATHScenario
 from helm.benchmark.scenarios.scenario import Input, Output, Reference
 
 
+@pytest.mark.skip(
+    reason="Incompatible with newer versions with diffusers>0.24.0. Fails with "
```

Comment:
Does this mean the MATH scenario itself also does not work if diffusers is installed? If so, please open an issue / add a TODO comment to fix this.